lib/corsica.ex

```diff
@@ -305,6 +311,12 @@ defmodule Corsica do
     |> maybe_update_option(:expose_headers, &Enum.join(&1, ", "))
   end
 
+  defp require_origins(%Options{origins: nil}) do
+    raise ArgumentError, message: "`:origins` option is required. It should a an array of domains or a string of a domain that you control. For example `origins: [\"https://app.example.com\", \"https://www.example.com\"]`. If you don't know which setting to use \"*\" will get you started, but it is VERY INSECURE. \"*\" completely disables all protections that CORS gives you, so you should never use \"*\" in a production environment. For more details see https://www.owasp.org/index.php/HTML5_Security_Cheat_Sheet#Cross_Origin_Resource_Sharing"
```

Comment:
Let's not put all this information in the error message. Let's use something like:

```elixir
raise ArgumentError, "the :origins option is required"
```

Folks should check the documentation in order to see which values are possible.